aggregator/election_mgr.go

```diff
@@ -496,8 +499,12 @@ func (mgr *electionManager) verifyPendingFollower(watch xwatch.Watch) {
 			}
 			return nil
 		}); verifyErr != nil {
-			mgr.logError("verify error", verifyErr)
-			continue
+			// If state has changed, we skip this stale change.
+			if !stateUnchangedFn() {
```

Comment:
That should never happen, because if it's evaluated as false, it means there is a new state set on the watchable, therefore incrementing the state id, so it should always be false afterwards.

Reply:
sounds good